hazelcast/src/main/java/com/hazelcast/instance/GroupProperties.java

```diff
@@ -375,6 +375,14 @@
     public static final String PROP_QUERY_MAX_LOCAL_PARTITION_LIMIT_FOR_PRE_CHECK
             = "hazelcast.query.max.local.partition.limit.for.precheck";
 
+    /**
+     * If enabled, the default serialization hierarchy (DataSerializable -> Portable -> Constant -> Custom )
+     * will be overrided and custom serialization will be checked first to serialize objects.
+     */
+    public static final String PROP_SERIALIZATION_CUSTOM_OVERRIDE
```

Comment:
No need for a public property imho. This is a hack to test certain behavior; we don't want to expose it to the outside world. Therefor it should not be added to GroupProperties; so this whole class can be reverted.